.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Type of change
+Please choose appropriate options.
+
+ Bug fix (non-breaking change which fixes an issue)
+ New feature (non-breaking change which adds functionality)
+ Breaking change (fix or feature that would cause existing functionality to not work as expected)
+ This change requires a documentation update
+How Has This Been Tested?
+Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+ Ran Test A
+ Ran Test B
+Test Configuration:
+
+Software versions:
+Hardware versions:
```

Comment:
Can you please update the default software and versions to build and run the services.